sbin/generate_grade_summaries.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+"""
+Script to trigger the generate grade summaries.
+
+Usage:
+./generate_grade_summaries.py <base_url> <token>
+"""
+
+import argparse
+import requests
+
+
+def main():
+    """Automatically call Generate Grade Summaries API."""
+    parser = argparse.ArgumentParser(
+        description='Automatically call Generate Grade Summaries API.'
+    )
+    parser.add_argument('base_url')
+    parser.add_argument('token')
+    args = parser.parse_args()
+
+    base_url = args.base_url.rstrip('/')
+    token = args.token
+
+    courses = list()
+    course_response = requests.get(
+        '{}/api/courses'.format(base_url),
+        headers={'Authorization': token}
+    )
+    if course_response.status_code == 200:
+        course_response = course_response.json()
+        if course_response["status"] == 'success':
+            courses = course_response['data']['unarchived_courses']
+        else:
+            print("ERROR: {}".format(course_response["message"]))
+            exit(-1)
+    else:
+        print("ERROR: Submitty Service Unavailable.")
+        exit(-1)
+
+    for course in courses:
+        grade_generation_response = requests.post(
+            '{}/api/{}/{}/reports/summaries'.format(
+                base_url, course['semester'], course['title']
+            ),
+            headers={'Authorization': token}
+        )
+        if grade_generation_response.status_code == 200:
+            grade_generation_response = grade_generation_response.json()
+            if grade_generation_response["status"] == 'success':
```

Comment:
@zjxiaohan 

Could you please change the wording of these lines to indicate that they generate grade summaries, not rainbow grades.